manifests/vsphere-7.0/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -0,0 +1,149 @@
+kind: Deployment
```

Comment:
Can we add a comment in the beginning with the supported k8s versions?

Reply:
done.